ios/Classes/MapboxMapController.swift

```diff
@@ -389,6 +389,7 @@ class MapboxMapController: NSObject, FlutterPlatformView, MGLMapViewDelegate, Ma
         if let channel = channel {
             channel.invokeMethod("map#onStyleLoaded", arguments: nil)
         }
+        _ = OfflineManager(mapview:mapView,registrar: registrar)
```

Comment:
Same question as on Android. What would happen with OfflineManager instance on rotation and calls?